client/domain/CodeEvalResultObjectFactorySpec.js

```diff
@@ -22,6 +22,7 @@ describe('CodeEvalResultObjectFactory', function() {
   var CODE = "code";
   var OUTPUT = "output";
   var CORRECTNESS_TEST_RESULTS = "correctnessTestResults";
+  var LAST_TASK_RESULTS = "s";
```

Comment:
Hi Sean. Thanks again for helping me with these tests! I'm looking into fixing it this morning.